components/Partials/Layout.js

```diff
@@ -17,6 +18,18 @@ export default function Layout({ children, togglePageActive }) {
       </Head>
       <Header togglePageActive={togglePageActive} />
       <StyledMain>{children}</StyledMain>
+      <StyledFooterImage
+        src="/images/footer-70.png"
+        sizes="100vw"
+        style={{
+          width: "100%",
+          height: "auto",
+        }}
```

Comment:
no inline styling, especially in components, that are already styled compoents :D